lib/ovh.js

```diff
@@ -654,6 +663,17 @@ var https = require('https'),
     return '$1$' + crypto.createHash('sha1').update(s.join('+')).digest('hex');
   };
 
+  /**
+   * Change the custom HTTPS agent used for reaching the OVH servers
+   *
+   * @param {Object} httpsAgent: the custom HTTPS agent
+   */
+  Ovh.prototype.setCustomHTTPSAgent = function (httpsAgent) {
```

Comment:
Constructor is "agent", internal is "httpsAgent", setter is "customHTTPSAgent". Please pick one :)